src/components/views/room_settings/ColorSettings.js

```diff
@@ -39,6 +39,9 @@ const ROOM_COLORS = [
     //["#595959", "#ececec"], // Grey makes everything appear disabled, so remove it for now
 ];
 
+// Dev note: this component is not attached anywhere, but is left here as it
```

Comment:
To me, it seems a bit strange to do this, since we can always revive anything from version control if we want it back. It makes me wonder if we have other orphaned components lying around as well...

Reply:
It's a lot harder to find once deleted, otherwise I'd generally agree. I have this component on my watch list: if we don't implement color settings, it'll disappear.

Reply:
Okay, as long as someone's keeping track of it, it seems fine.